layouts/partials/events/person.html

```diff
@@ -4,7 +4,7 @@
     <div class="card-content">
```

Comment:
This is great! Thank you for fixing https://github.com/cncf/kubernetes-community-days/issues/268. Please submit this in a separate PR rather than requesting these fixes within this PR. Thank you!

Reply:
This is already fixed. It needs a rebase